docs/usage/fields.md

```diff
@@ -191,7 +191,7 @@ print(user.model_dump(by_alias=True))  # (2)!
     class User(BaseModel):
         model_config = ConfigDict(populate_by_name=True)
 
-        name: str = Field(..., alias='username')
+        name: str = Field(..., alias=str('username'))
```

Comment:
we don't need this change.

What is the reason for this change here?

Reply:
If that is not required, then this part of the same document is wrong then?
> To "trick" VSCode into preferring the field name, you can use the `str` function to wrap the alias value

I found it weird that there are two code blocks essentially equal to each other and the code does not matches the description above it